From 4e4335b10b1c38ff8f852f6916f787677712a0d6 Mon Sep 17 00:00:00 2001
From: Paul Gortmaker <paul.gortmaker@windriver.com>
Date: Mon, 23 Oct 2017 12:42:37 -0400
Subject: [PATCH] mfd: fix implicit dependency on memset

We see this on an -rt version, but that doesn't change the fact
that there is an implicit include here:

drivers/mfd/atmel-smc.c: In function 'atmel_smc_cs_conf_init':
drivers/mfd/atmel-smc.c:24:2: error: implicit declaration of function 'memset' [-Werror=implicit-function-declaration]
drivers/mfd/atmel-smc.c:24:2: warning: incompatible implicit declaration of built-in function 'memset' [enabled by default]

Fix it for everyone by explicitly adding the include for the memset
call just a few lines below.

Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>
---
 drivers/mfd/atmel-smc.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/mfd/atmel-smc.c b/drivers/mfd/atmel-smc.c
index 954cf0f66a31..c2f7069d537c 100644
--- a/drivers/mfd/atmel-smc.c
+++ b/drivers/mfd/atmel-smc.c
@@ -11,6 +11,7 @@
  * published by the Free Software Foundation.
  */
 
+#include <linux/string.h>
 #include <linux/mfd/syscon/atmel-smc.h>
 
 /**
-- 
2.14.1

